src/main/java/cubicchunks/asm/CubicChunksMixinConfig.java

```diff
@@ -144,5 +153,46 @@ public static String getNicelyFormattedName(String name) {
         }
         return out.toString();
     }
-
+    
+    private void writeConfigToJson(File configFile) throws IOException {
+        JsonWriter writer = new JsonWriter(new FileWriter(configFile));
+        writer.setIndent(" ");
+        writer.beginArray();
+        for (BoolOptions configOption : BoolOptions.values()) {
+            writer.beginObject();
+            writer.name("name");
+            writer.value(configOption.name());
+            writer.name("description");
```

Comment:
Looks like Gson supports comments, but there doesn't seem to be a way to write them using the writer. I knowthat json specification doesn't allow them. Then leave the description the way it is, but name and value shoudl be one thing, it makes no sense to do what you did here.